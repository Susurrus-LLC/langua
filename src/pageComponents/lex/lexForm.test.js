import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import LexForm from './lexForm'

describe('LexForm', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<LexForm />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately', () => {
    let tree = renderer.create(<LexForm />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
