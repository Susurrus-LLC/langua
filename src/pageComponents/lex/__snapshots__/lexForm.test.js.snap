// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LexForm renders accurately 1`] = `
<form
  className="form"
  name="lex-form"
/>
`;
